src/WinRT.Runtime/Projections/Nullable.cs

```diff
@@ -2418,6 +2446,53 @@ unsafe object IWinRTNullableTypeDetails.GetNullableValue(IInspectable inspectabl
                 Marshal.Release(nullablePtr);
             }
         }
+
+        // Delegates are handled separately.
+        Type IWinRTNullableTypeDetails.GetNullableType() => throw new NotImplementedException();

```

Comment:
Nit: should this be `NotSupportedException`?

Reply:
We do support getting a `Nullable` like type for delegates just in case the WinUI native side asks for it given in WinRT delegates still support `IReference`.  We use our own type to represent it rather than actually `Nullable` which doesn't support it.  It is handled elsewhere rather than here which is why this is `NotImplementedException` rather than saying it is not supported.